src/Trace/README.md

```diff
@@ -1,115 +1,65 @@
-RequestTracer::start($reporter);
-```
-
-## Customizing
-
-### Reporting Traces
-
-The above sample uses the `AsyncReporter` to report trace results to the Stackdriver servers.
-If you are using the experimental
-[google-cloud-batch daemon](https://github.com/GoogleCloudPlatform/google-cloud-php-core/blob/master/Batch/BatchDaemon.php)
-and have set the `IS_BATCH_DAEMON_RUNNING=true` environment variable, then the reporting of the trace will happen
-asynchronously. If not, the reporting will happen at the end of the request and can add some latency to your request.
-
-For testing/development, we also provide an `EchoReporter`, `FileReporter` and `LoggerReporter`.
-
-If you would like to provide your own reporter, create a class that implements `ReporterInterface`.
-
-### Sampling Rate
-
-By default we attempt to trace all requests. This is not ideal as a little bit of
-latency and require more memory for requests that are traced. To trace a sampling
-of requests, configure a sampler.
+This library contains code to interact with the Stackdriver Trace V2 API.
 
-The preferred sampler is the `QpsSampler` (Queries Per Second). This sampler implementation
-requires a PSR-6 cache implementation to function.
+### Creating a Trace
 
 ```php
 use Google\Cloud\Trace\TraceClient;
-use Google\Cloud\Trace\Reporter\SyncReporter;
-use Google\Cloud\Trace\Sampler\QpsSampler;
-
-$trace = new TraceClient();
-$reporter = new SyncReporter($trace);
-$cache = new SomeCacheImplementation();
-$sampler = new QpsSampler($cache, ['rate' => 0.1]); // sample 0.1 requests per second
-RequestTracer::start($reporter, ['sampler' => $sampler]);
-```
-
-Please note: While required for the `QpsSampler`, a PSR-6 implementation is
-not included in this library. It will be necessary to include a separate
-dependency to fulfill this requirement. For PSR-6 implementations, please see the
-[Packagist PHP Package Repository](https://packagist.org/providers/psr/cache-implementation).
-If the APCu extension is available (available on Google AppEngine Flexible Environment)
-and you include the cache/apcu-adapter composer package, we will set up the cache for you.
-
-You can also choose to use the `RandomSampler` which simply samples a flat
-percentage of requests.
 
-```php
-use Google\Cloud\Trace\TraceClient;
-use Google\Cloud\Trace\Reporter\SyncReporter;
-use Google\Cloud\Trace\Sampler\RandomSampler;
+$client = new TraceClient();
+$trace = $client->trace();
+$span = $trace->span(['name' => 'main']);
+$trace->setSpans([$span]);
 
-$trace = new TraceClient();
-$reporter = new SyncReporter($trace);
-$sampler = new RandomSampler(0.1); // sample 10% of requests
-RequestTracer::start($reporter, ['sampler' => $sampler]);
+$client->insert($trace);
 ```
 
-If you would like to provide your own sampler, create a class that implements `SamplerInterface`.
+### Using OpenCensus
 
-## Tracing Code Blocks
+We highly recommend using the [OpenCensus][opencensus] project to instrument
```

Comment:
fwiw, a brief statement of what OpenCensus is and why it should be preferred over direct use of this client could be helpful.

Reply:
Added a note about how OpenCensus will maintain the integrations with frameworks and tools.